docs/release-notes/1.1/release-1.1.md

```diff
@@ -0,0 +1,208 @@
+
+<table>
+    <thead>
+        <tr>
+            <th>Oldest compatible .NET for Apache Spark release version</th>
+        </tr>
+    </thead>
+    <tbody align="center">
+        <tr>
+            <td>v1.0.0</td>
+        </tr>
+    </tbody>
+</table>
+
+### Supported Spark Versions
+
+The following table outlines the supported Spark versions along with the microsoft-spark JAR to use with:
+
+<table>
+    <thead>
+        <tr>
+            <th>Spark Version</th>
+            <th>microsoft-spark JAR</th>
+        </tr>
+    </thead>
+    <tbody align="center">
+        <tr>
+            <td>2.3.*</td>
+            <td>microsoft-spark-2-3_2.11-1.1.0.jar</td>
+        </tr>
+        <tr>
+            <td>2.4.0</td>
+            <td rowspan=7>microsoft-spark-2-4_2.11-1.1.0.jar</td>
+        </tr>
+        <tr>
+            <td>2.4.1</td>
+        </tr>
+        <tr>
+            <td>2.4.3</td>
+        </tr>
+        <tr>
+            <td>2.4.4</td>
+        </tr>
+        <tr>
+            <td>2.4.5</td>
+        </tr>
+        <tr>
+            <td>2.4.6</td>
+        </tr>
+        <tr>
+            <td>2.4.7</td>
+        </tr>
+        <tr>
+            <td>2.4.2</td>
+            <td><a href="https://github.com/dotnet/spark/issues/60">Not supported</a></td>
+        </tr>
+        <tr>
+            <td>3.0.0</td>
+            <td rowspan=3>microsoft-spark-3-0_2.12-1.1.0.jar</td>
+        </tr>
+        <tr>
+            <td>3.0.1</td>
+        </tr>
+		<tr>
+            <td>3.0.2</td>
+        </tr>
+    </tbody>
+</table>
+
+### Supported Delta Versions
+
+The following table outlines the supported Delta versions along with the Microsoft.Spark.Extensions version to use with:
+
+<table>
+    <thead>
+        <tr>
+            <th>Delta Version</th>
+            <th>Microsoft.Spark.Extensions.Delta</th>
+        </tr>
+    </thead>
+    <tbody align="center">
+        <tr>
+            <td>0.1.0</td>
+            <td rowspan=9>1.1.0</td>
+        </tr>
+        <tr>
+            <td>0.2.0</td>
+        </tr>
+        <tr>
+            <td>0.3.0</td>
+        </tr>
+        <tr>
+            <td>0.4.0</td>
+        </tr>
+        <tr>
+            <td>0.5.0</td>
+        </tr>
+        <tr>
+            <td>0.6.0</td>
+        </tr>
+        <tr>
+            <td>0.6.1</td>
+        </tr>
+        <tr>
+            <td>0.7.0</td>
+        </tr>
+		<tr>
+            <td>0.8.0</td>
+        </tr>
+    </tbody>
+</table>
+
+### Supported Hyperspace Versions
+
+The following table outlines the supported Hyperspace versions along with the Microsoft.Spark.Extensions version to use with:
+
+<table>
+    <thead>
+        <tr>
+            <th>Hyperspace Version</th>
+            <th>Microsoft.Spark.Extensions.Hyperspace</th>
+        </tr>
+    </thead>
+    <tbody align="center">
+        <tr>
+            <td>0.1.0</td>
+            <td rowspan=3>1.1.0</td>
+        </tr>
+        <tr>
+            <td>0.2.0</td>
+        </tr>
```

Comment:
was 0.3.0 never supported ?

Reply:
Looks like https://github.com/dotnet/spark/pull/815 includes Attributes that indicate apis introduced in 0.0.3 so I think it is supported ?

Reply:
Looks that way, I didn't find a PR for 0.3.0. @AFFogarty ?

Reply:
Got it, added 0.3.0 support